.github/workflows/lint.yml

```diff
@@ -23,7 +23,7 @@ jobs:
             **/**.go
             go.mod
             go.sum
-      - uses: golangci/golangci-lint-action@v3.7.0
+      - uses: golangci/golangci-lint-action@v4.0.0
```

Comment:
The v4.0.0 release notes don't contain any breaking changes: https://github.com/golangci/golangci-lint-action/releases/tag/v4.0.0

Created: https://github.com/golangci/golangci-lint-action/issues/974

but this is probably safe to merge considering golangci-lint passed.